### ISSUE TITLE
Very slow qt widgets in Spyder ipython console.

### ISSUE THREAD

**Issue by jaxankey** [spyder-ide/spyder#13399]
## Problem Description

Today, after a conda update (and then a full Anaconda reinstall), my QtWidgets have become incredibly slow when launching from the spyder ipython console, taking a second or so to update when I click around. This never was the case before. By contrast, executing in a system terminal with the default python (or ipython), it is all the usual level of fast and responsive.

### What steps reproduce the problem?

Here is a test script (sorry, it relies on pyqtgraph to get at the Qt library):
```Python
# Try to run the qt magic to take it out of inline mode.
# This is required to avoid a freeze.
try:    get_ipython().run_line_magic('matplotlib', 'qt')
except: pass # Non-ipython terminal.

# Get access to Qt4 API (this is still a Qt5 environment, however)
import pyqtgraph
QtGui  = pyqtgraph.QtGui
QtCore = pyqtgraph.QtCore

# Make the app.
_qtapp = QtCore.QCoreApplication.instance()
if not _qtapp: _qtapp = QtGui.QApplication([])


# Create a button and show it.
b = QtGui.QPushButton('Pants')
b.setGeometry(0,0,200,200)
b.show()
```

### What is the expected output? What do you see instead?

This pops up a button as usual, but when I click and hold, it takes a second or so to switch to the "pushed" state, and then another second to "unpush" after I've released my mouse button. Again, in other terminals outside of spyder (and before today), the response is immediate.

### Paste Traceback/Error Below (if applicable)

No errors in traceback.

## Versions

or (if Spyder won't launch) the "conda list" command
from the Anaconda Prompt/Terminal/command line. --->

* Spyder version: 4.1.4
* Python version: 3.7
* Qt version: 5.9.7
* PyQt version: 5.9.2
* PyQtGraph verions: 0.11.0
* Operating System name/version: Linux Mint 20

### Dependencies

```
# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.5.0 (OK)
diff_match_patch >=20181111    :  20200713 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.16.1 (OK)
jedi =0.17.1                   :  0.17.1 (OK)
keyring                        :  None (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  1.1.0 (OK)
parso =0.7.0                   :  0.7.0 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=1.0                   :  2.5.3 (OK)
pyls >=0.34.0;<1.0.0           :  0.34.1 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.2 (OK)
qtconsole >=4.6.0              :  4.7.5 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.1.2 (OK)
spyder_kernels >=1.9.2;<1.10.0 :  1.9.2 (OK)
watchdog                       :  None (OK)
xdg >=0.26                     :  0.26 (OK)
zmq >=17                       :  19.0.1 (OK)

# Optional:
cython >=0.21                  :  None (OK)
matplotlib >=2.0.0             :  3.2.2 (OK)
numpy >=1.7                    :  1.18.5 (OK)
pandas >=0.13.1                :  None (OK)
scipy >=0.17.0                 :  1.5.0 (OK)
sympy >=0.7.3                  :  None (OK)
```

**Comment by ccordoba12** [comment]
Hi @jaxankey, thanks for reporting. Please update to ipykernel 5.3.4, that should fix your error.

If that version is not yet available in Anaconda, please downgrade to 5.3.2.

**Comment by jaxankey** [comment]
Perfect, thank you! I hope someone is leaning on Conda about this...
